v63002/outref/gtm8717.txt

```diff
@@ -0,0 +1,6 @@
+# Testing 1000 random variations of $select syntax, checking for FATAL errors
```

Comment:
Add a line saying we have seen some of these random variations generate GTMASSERT2 and KILLBYSIGSINFO1 fatal errors using V63001A